backend/models/user.py

```diff
@@ -3,35 +3,48 @@
 from typing import Optional, List
 from pydantic import BaseModel, validator, constr
 
+
+class UserRole(str, Enum):
+    ANGANVADI = 'ANGANVADI'
+    NGO_MEMBER = 'NGO_MEMBER'
+    RELATIVE = 'RELATIVE'
+
+
 class UserType(str, Enum):
     PREGNANT = 'PREGNANT'
     LACTATING = 'LACTATING'
     CAREGIVER = 'CAREGIVER'
 
+
 class GenderType(str, Enum):
     MALE = 'MALE'
     FEMALE = 'FEMALE'
 
+
 class Child(BaseModel):
     name: str
     gender: GenderType
     dob: datetime.date
+
     @validator("dob", pre=True)
     def parse_dob(cls, value):
         return datetime.datetime.strptime(
             value,
             "%d/%m/%Y"
         ).date()
 
+
 class CreateUserSchema(BaseModel):
     name: str
     phone_number: constr(
         strip_whitespace=True,
         regex=r"^(\+)[1-9][0-9\-\(\)\.]{9,15}$",
     )
     user_type: UserType
+    user_role: UserRole
```

Comment:
This should be optional field. There can be a case where user decided to not select "is creating for someone else", in that case userRole field should be empty/blank.